GeorgianMemenator/Scenes/DogBreedDetail/DogBreedDetailViewModel.swift

```diff
@@ -4,9 +4,23 @@
 //
 //  Created by Qeti Avanashvili on 22.03.23.
 //
-
+import SwiftUI
 import Foundation
+import Combine
 
 class DogBreedDetailViewModel: ObservableObject {
+    @Published var dogBreedDetail: DogBreed?
+    
+    private let dogBreedDetailService: DogBreedsServing
+    private let dogBreedId: String
     
+    init(dogBreedDetailService: DogBreedsServing, dogBreedId: String) {
+        self.dogBreedDetailService = dogBreedDetailService
+        self.dogBreedId = dogBreedId
+    }
+   
+    func getDogBreedDetail() {
+        dogBreedDetailService
```

Comment:
This function needs the to `sink` and update the dog breed detail whenever its fetched. Currently you do nothing, please see how you have implemented service function in the list vm.

Reply:
okey, I'll try now